WordPress/Classes/ViewRelated/System/Notices/UntouchableWindow.swift

```diff
@@ -0,0 +1,76 @@
+@objc class UntouchableWindow: UIWindow {
+    let untouchableViewController: UntouchableViewController
+
+    override init(frame: CGRect) {
+        untouchableViewController = UntouchableViewController()
```

Comment:
Not a big deal but what do you think if we set the property value part of the property’s declaration?